### PR TITLE
WIP: [gallery] Add button for informations about media

### PR DESCRIPTION
**Pull request by vknecht**
Use libexif for image metadata, and Qt for videos
through QMediaObject and QMediaMetadata.

### CODE REVIEW

**Comment by vknecht** [comment]
I could use some tips about Qt formatting in AboutMedia.qml, where word-wrapping both labels lengths instead of the value one... any idea ?

**Comment by piggz** [comment]
Is this still WIP?

**Comment by vknecht** [comment]
> Is this still WIP?

It's stalled, previous comment about line-length/word-wrapping still applies.
Other than that it should work fine.

**Comment by Karry** [comment]
And what about using Sailfish component `DetailItem`https://github.com/Karry/harbour-advanced-camera/commit/57c7fd495468bb3337b44eccd6283399eb1620f9

Snimek_20210702_001

File: src/exifmodel.cpp [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    m_lib.setFileName("libexif.so.12");
+    if (m_lib.load())
+    {
+        f_fromfile =  (ExifFromFile) m_lib.resolve("exif_data_new_from_file");
+        f_getvalue = (ExifGetValue) m_lib.resolve("exif_entry_get_value");
+        f_foreachentry = (ExifForEachEntry) m_lib.resolve("exif_content_foreach_entry");
+        f_foreachcontent = (ExifForEachContent) m_lib.resolve("exif_data_foreach_content");
+        f_getifd = (ExifGetIfd) m_lib.resolve("exif_content_get_ifd");
+        f_gettag = (ExifGetTag) m_lib.resolve("exif_tag_get_name_in_ifd");
+        f_dump = (ExifDump) m_lib.resolve("exif_data_dump");
```
Comment (Karry) [inline src/exifmodel.cpp:43]: why so complicated? would not be better to link binary with libexif and use functions directly?

**Review by Karry — COMMENTED** [review]
I like the feature, just don't understand why libexif is loaded dynamically?

Moreover, it would be great to display image histogram in future ;-)

**Comment by piggz** [comment]
It will be because of harbour restrictions.

**Comment by Karry** [comment]
> It will be because of harbour restrictions.

Damn, I thought that Advanced Camera is just on OpenRepos...

**Pull request closed without merging**